cmd/allocator/main.go

```diff
@@ -277,8 +277,8 @@ func (h *serviceHandler) Allocate(ctx context.Context, in *pb.AllocationRequest)
 		logger.Errorf("internal server error - Bad GSA format %v", resultObj)
 		return nil, status.Errorf(codes.Internal, "internal server error- Bad GSA format %v", resultObj)
 	}
-	response := converters.ConvertGSAV1ToAllocationResponseV1Alpha1(allocatedGsa)
-	logger.WithField("response", response).Infof("allocation response is being sent")
+	response, err := converters.ConvertGSAV1ToAllocationResponseV1Alpha1(allocatedGsa)
+	logger.WithField("response", response).WithField("err", err).Infof("allocation response is being sent")
```

Comment:
Could we use https://godoc.org/gopkg.in/Sirupsen/logrus.v0#WithError here?
```suggestion
	logger.WithField("response", response).WithError(err).Infof("allocation response is being sent")
```

Reply:
Thanks. Done.